### PR TITLE
switch to the main branch

### PR DESCRIPTION
**Pull request by kdorheim**
This is to address #697 to change from the default branch from being master to main. To do this will need to update the github action files and documentation.

### CODE REVIEW

**Comment by kdorheim** [comment]
@bpbond and @crvernon this should address the code changes fro #697 , I think. I think the outstanding TODOs would be to change the default branch on github and then the default branch on code cov. Do we do that now? Or wait until this is merged into master and then update? Does it need a new release associated with that change? What do you think?

File: .github/workflows/R-CMD-check.yaml
Comment (bpbond) [inline .github/workflows/R-CMD-check.yaml:6]: Or do we want to just add `- main` for now, do the branch switch, ensure everything working, and then finally delete the `- main` line?

**Review by bpbond — APPROVED** [review]
This seems fine; thanks @kdorheim

**Comment by bpbond** [comment]
@kdorheim I'm not sure of the best sequence here, but I guess I would merge this into master and then update the branch name. Don't see the need for a new release since there's no code change.

**Comment by kdorheim** [comment]
> @kdorheim I'm not sure of the best sequence here, but I guess I would merge this into master and then update the branch name. Don't see the need for a new release since there's no code change.

Yea I am also at a loss here 🤷‍♀️

**Pull request merged**
